gokart/task.py

```diff
@@ -176,7 +173,8 @@ def make_target(self, relative_file_path: Optional[str] = None, use_unique_id: b
                                          redis_host=self.redis_host,
                                          redis_port=self.redis_port,
                                          redis_timeout=self.redis_timeout,
-                                         raise_task_lock_exception_on_collision=self.raise_task_lock_exception_on_collision)
+                                         raise_task_lock_exception_on_collision=False)
```

Comment:
@mski-iksm Plz describe why we need to keep  `raise_task_lock_exception_on_collision` for make_redis_params while this is always set to be False

Reply:
@Hi-king 
In the coming feature, TaskOnKart.run() will be wrapped with task collision lock to prevent running same task at the same time.
I want to keep RedisParams.raise_task_lock_exception_on_collision to be used in this new feature.

Reply:
@mski-iksm Thx for clarifying :)